src/Server.UI/Pages/Objectives/Tasks/CompleteTaskDialog.razor

```diff
@@ -20,6 +20,10 @@
                           Lines="5"
                           Class="mt-4" />
         </MudForm>
+        <MudAlert Class="mt-4" Severity="Severity.Info">
+            No further payables (Activity/ETE/PSF) can be claimed for this task once completed. 
+            You may still record payables against other tasks.
```

Comment:
Now I look back, I don't think usage of the word "payables" here is appropriate. I'll ask the question re wording for this alert.